src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.wasm.cs

```diff
@@ -32,13 +33,29 @@ internal Size ScrollBarSize
 
 		public ScrollContentPresenter()
 		{
-			PointerReleased += ScrollViewer_PointerReleased;
-			PointerPressed += ScrollViewer_PointerPressed;
-			PointerCanceled += ScrollContentPresenter_PointerCanceled;
-			PointerMoved += ScrollContentPresenter_PointerMoved;
-			PointerEntered += ScrollContentPresenter_PointerEntered;
-			PointerExited += ScrollContentPresenter_PointerExited;
-			PointerWheelChanged += ScrollContentPresenter_PointerWheelChanged;
+		}
+
+		private void TryRegisterEvents(ScrollBarVisibility visibility)
+		{
+
+			if (
+				!_eventsRegistered
+				&& (visibility == ScrollBarVisibility.Auto || visibility == ScrollBarVisibility.Visible))
```

Comment:
Wondering if we should not also register for `ScrollBarVisibility.Disabled`: This mode would also render the scrollbar but in disabled state.